xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -316,7 +318,11 @@ For more information, see [How to serialize and deserialize JSON](/dotnet/standa
 
 -or-
 
-There is remaining data in the string beyond a single JSON value.</exception>
+There is remaining data in the string beyond a single JSON value.
+
+-or-
+
+.NET Core 3.1 and later versions only: <typeparamref name="TValue" /> is <see cref="T:System.Char" /> and <paramref name="json" /> is a null reference (<code>JsonSerializer.Deserialize&lt;char&gt;("null");</code>) or an empty string.</exception>
```

Comment:
This is unnecessary info to the user, and isn't accurate. Please revert this part of the change.